internal/pkg/cli/options/options.go

```diff
@@ -34,6 +45,7 @@ func New() *Options {
 	envNaming := env.NewNamingConvention()
 	return &Options{
 		envNaming: envNaming,
+		setBy:     make(map[string]SetBy),
```

Comment:
Potrebujem nejaky mechanizmus, aby som vedel zistit, ci bola hodnota nastavena z flag/env alebo dialogu.
Pouzita `viper` kniznica to nevie, tak si to musim napisat sam, pomocou tejto map.